docs/administration/users.md

```diff
@@ -14,33 +14,42 @@ Roles are used in the platform to grant the given groups with some **capabilitie
+| &nbsp;&nbsp;&nbsp;&nbsp;`Delete knowledge`                     | Delete entities and relationships.                                                      |
+| &nbsp;&nbsp;&nbsp;&nbsp;`Manage authorized members`            | Restrict the access to an entity to a user, group or organization.                      |
+| &nbsp;&nbsp;&nbsp;&nbsp;`Bypass enforced reference`            | If external references enforced in a type of entity, be able to bypass the enforcement. |
+| &nbsp;&nbsp;`Upload knowledge files`                           | Upload files in the `Data` and `Content` section of entities.                           |
+| &nbsp;&nbsp;`Import knowledge`                                 | Trigger the ingestion of an uploaded file.                                              |
+| &nbsp;&nbsp;`Download knowledge export`                        | Download the exports generated in the entities (in the `Data` section).                 |
+| &nbsp;&nbsp;&nbsp;&nbsp;`Generate knowledge export`            | Trigger the export of the knowledge of an entity.                                       |
+| &nbsp;&nbsp;`Ask for knowledge enrichment`                     | Trigger an enrichment for a given entity.                                               |
+| `Access Dashboards and investigations`                         | Access to existing custom dashboards and investigations.                                |
+| &nbsp;&nbsp;`Create / Update dashboards and investigations`    | Create and update custom dashboards and investigations.                                 |
+| &nbsp;&nbsp;&nbsp;&nbsp;`Delete dashboards and investigations` | Delete existing custom dashboards and investigations.                                   |
+| &nbsp;&nbsp;&nbsp;&nbsp;`Manage public dashboards`             | Manage public dashboards                                                                |
+| `Access connectors`                                            | Read information in the `Data > Connectors` section.                                    |
+| &nbsp;&nbsp;`Manage connector state`                           | Reset the connector state to restart ingestion from the beginning.                      |
+| `Connectors API usage: register, ping, export push ...`        | Connectors specific permissions for register, ping, push export files, etc.             |
+| `Access data sharing`                                          | Access and consume data such as TAXII collections, CSV feeds & live streams.            |
+| &nbsp;&nbsp;`Manage data sharing`                              | Share data such as TAXII collections, CSV feeds & live streams or custom dashboards.    |
+| `Access ingestion`                                             | Access (read only) Remote OCTI streams TAXII feeds, RSS Feeds, CSV Feeds.               |
```

Comment:
```suggestion
| `Access ingestion`                                             | Access (read only) remote OCTI streams TAXII feeds, RSS feeds, CSV feeds.               |
```
it's either all with capital `F` or all with `f`

Reply:
Done